src/Resources/doc/annotations/routing.rst

```diff
@@ -29,6 +29,22 @@ annotation, so you only have to update the annotation class namespace:
         }
     }
 
+Alternatively, it can be done with the help of the PHP 8 attribute:
+
+.. code-block:: diff
+
+    -use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+    +use Symfony\Component\Routing\Annotation\Route;
+
+    class DefaultController extends Controller
+    {
+        #[Route('/')]
```

Comment:
We should render the old route as a Doctrine annotation, shouldn't we?

Reply:
I'm not sure about it. If it's so, we just need to remove this block at all. Please advise :)

Reply:
I mean, for the other example for the migration to Symfony's `Route` annotation, this is what you do. But I think, it would be just fine if we ignored attributes for the migration completely.

Reply:
@derrabus got you. I've just updated it to make consistent with the other blocks. Could it be done this way?